ourjapanlife/findadoc-web
# Diego/mod home top bar

Resolves #[Issue number]

## 🔧 What changed

- Top Bar on the submission panel in the initial view
   - Created a component called ModDashboardTopBar to hold the top bar in the dashboard of the submission panel.
   - The input console.logs the input when Enter is pressed or the Magnifying glass is clicked.
   - Both the TopBar text and the placeholder are linked in the multi-language functionality

- Other changes
   - Added some vertical padding in the left bar to look closer to the figma. (I can remove that if it was unnecessary).

## 🧪 Testing instructions

1. Access the submission panel
2. Observe the top bar
3. Change the language between japanese or english. Note the change in the text from the left and in the placeholder.
4. Type any text in the input.
5. Press Enter
6. Observe the same text being shown in the console.
7. Click in the Magnifying glass.
8. Observe the text being shown in the console.

## 📸 Screenshots


-   ### Before
![image](https://github.com/ourjapanlife/findadoc-web/assets/92993129/ebc3ba73-0c66-4eb3-9385-d2a2abd1c5ce)

-   ### After
![image](https://github.com/ourjapanlife/findadoc-web/assets/92993129/4dad4d3c-1788-44e0-9b13-a0dacd024a20)
